pkg/http-handler/http-handler.go

```diff
@@ -377,6 +377,7 @@ func SetupDefaultRoutes(r *mux.Router, namespaceHeaderName, namespaceHeaderPatte
 	if err != nil {
 		return err
 	}
+	//TODO(agamez): move these endpoints to a separate plugin when possible
```

Comment:
Yes, though hopefully we won't just move them. For example, we don't want to provide generic endpoints that can return all app repositories. Let's see.